feat(tag): comma-separated `--level` (e.g. `--level pipeline,job`) (#2388)
DataDog/datadog-ci: packages/base/src/commands/tag/README.md

datadog-ci tag --level job --tags "go.version:`go version`"
```

> - `--level` is one or more of `[pipeline, job, stage, step]`. It determines in what span(s) the tags will be added.

Reviewer: Now that `--level` can take multiple values, users could be tempted to do something like `--title '[stage, step]'`, it may be confusing. I think `{...}` reads more like a set of values.
```suggestion
- `--level` is a comma-separated list of `{pipeline,job,stage,step}`. It determines in what span(s) the tags will be added.
```

>   If `pipeline` is selected then the tags will be added to the pipeline trace span. If `job` is selected it will be added

Reviewer: **nit** Let's move this to a new paragraph
```suggestion

  If `pipeline` is selected then the tags will be added to the pipeline trace span. If `job` is selected it will be added
```

  to the span for the currently running job. If `stage` is selected it will be added to the span for the currently
  running stage. If `step` is selected it will be added to the span for the currently running step.

>   The value is **additive**: pass a comma-separated list to add the same tag set to several spans at once, e.g.

Reviewer: **nit** let's emphasize it's the multiple **levels** that are interesting
```suggestion
    Pass a comma-separated list to add the same tags to multiple levels at once, e.g.
```
